templates/zerver/dev_tools.html

```diff
@@ -0,0 +1,43 @@
+{% extends "zerver/portico.html" %}
+
+{# Login page. #}
+{% block portico_content %}
+<div id="devtools-page">
+    <h2>Useful development URLs</h2>
+    <p>
+        {% trans %}
+        Below is a list of useful URLs that provide information useful in development.
+        Most of these require you to run a command to build the relevant content. This
+        table specifies which command to use for each page. Make sure your development server
+        is still running when you visit these!
+        {% endtrans %}
```

Comment:
We generally don't tag for translation the development environment specific pages, just to save translations low-value translations.